lib/web/controllers/webhook_controller.ex

```diff
@@ -138,21 +138,29 @@ defmodule BorsNG.WebhookController do
     project = Repo.get_by!(Project, repo_xref: repo_xref)
     pr = BorsNG.GitHub.Pr.from_json!(conn.body_params["pull_request"])
     patch = Syncer.sync_patch(project.id, pr)
+    is_draft = pr["draft"]
 
-    do_webhook_pr(conn, %{
-      action: conn.body_params["action"],
-      project: project,
-      patch: patch,
-      author: patch.author,
-      pr: pr
-    })
+    if !is_draft do
+      do_webhook_pr(conn, %{
+        action: conn.body_params["action"],
+        project: project,
+        patch: patch,
+        author: patch.author,
+        pr: pr
+      })
+    end
```

Comment:
```suggestion
      do_webhook_pr(conn, %{
        action: conn.body_params["action"],
        project: project,
        patch: patch,
        author: patch.author,
        pr: pr,
        is_draft: is_draft
      })
```

Then you can create a a function:

```elixir
do_webhook_pr(_conn, %{is_draft: true}) do Logger... end
```

But totally optional. 😄